package.json

```diff
@@ -11,7 +11,7 @@
     "build:web": "expo export -p web",
     "lint": "npx tsx packages/scripts/paralint",
     "lint-fix": "eslint --cache --ext .js,.jsx,.ts,.tsx . --fix",
-    "unused-exports": "ts-unused-exports ./tsconfig.json --excludePathsFromReport=\"packages/api;packages/contracts-clients;packages/evm-contracts-clients;packages/screens/RiotGame/RiotGameBridgeScreen.tsx;packages/components/socialFeed/RichText/inline-toolbar;app.config.js;weshd;./App.tsx;packages/modules/FileSystem/index.ts;.*\\.web|.electron|.native|.d.ts\"",
+    "unused-exports": "ts-unused-exports ./tsconfig.json --excludePathsFromReport=\"packages/api;packages/contracts-clients;packages/evm-contracts-clients;packages/screens/RiotGame/RiotGameBridgeScreen.tsx;packages/components/socialFeed/RichText/inline-toolbar;app.config.js;weshd;./App.tsx;packages/modules/FileSystem/index.ts;packages/screens/Mini/Wallet/MiniWalletScreen.tsx;packages/screens/Mini/Feed/MiniFeedScreen.tsx;packages/screens/Mini/Chat/MiniChatScreen.tsx;.*\\.web|.electron|.native|.d.ts\"",
```

Comment:
I understand why this is necessary but it feels "wrong".
What do you think @n0izn0iz ?

Reply:
why is this necessary?